types/k6/experimental/browser/locator.d.ts

```diff
@@ -8,6 +8,7 @@ import {
     EvaluationArgument,
     ElementState,
     MouseMultiClickOptions,
+    StrictnessOptions,
```

Comment:
`Locator` APIs shouldn't be able to override the `strict` option, and it's always `true` for its methods.

Reply:
I agree, but similar to my [comment on `ElementHandle.uncheck()` I'd suggest that we match the current implementation, file a bug in `xk6-browser` and update the type definition once it's fixed.